lib/report_portal/settings.rb

```diff
@@ -46,18 +46,28 @@ def initialize
         'use_standard_logger' => false,
         'launch_id' => false,
         'file_with_launch_id' => false,
+        'launch_uuid'=>true #used when multiple cucumber processes executed 
       }
 
       keys.each do |key, is_required|
         define_singleton_method(key.to_sym) { setting(key) }
         fail "ReportPortal: Define environment variable '#{PREFIX}#{key}' or key #{key} in the configuration YAML file" if is_required && public_send(key).nil?
       end
+      launch_uuid = SecureRandom.uuid unless launch_uuid
```

Comment:
`launch_uuid` is marked as a required parameter. But here it's set to some value if it's not present. How is that possible?